synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -58,6 +58,7 @@
 
 #endif
 
+#include <synfig/os.h>
```

Comment:
Move this line up, right above this line : `#include <synfig/rendering/renderer.h>`

We try to group #includes and sort them alphabetically.

Reply:
@rodolforg the master branch had some imports which werent in alphabetical order and also it was causing conflicts with my pr so i put in os.h in the empty line to resolve it